viite-UI/src/model/RoadCollection.js

```diff
@@ -331,12 +300,12 @@
         projectId: projectId
       }, function (fetchedLinks) {
         var projectLinks = _.chain(fetchedLinks).flatten().filter(function (link) {
-          return link.status === LinkStatus.NotHandled.value
-              || link.status === LinkStatus.New.value
-              || link.status === LinkStatus.Terminated.value
-              || link.status === LinkStatus.Unchanged.value
-              || link.status === LinkStatus.Numbering.value
-              || link.status === LinkStatus.Transfer.value;
+          return link.status === LinkStatus.NotHandled.value ||
```

Comment:
Hmm, I think || would be clearer in the beginning of the row.
Murmuring to this change.

Linter rules were the reason for this I heard - can the linter rules be tweaked to allow (or to encourage) to use these at the beginning of the row?

Reply:
tämän säännön muuttaminen vaatisi kaikkien tämän tyylisten kohtien muuttamista jotta varoituksista/erroreista päästäisi eroon. Nopealla testauksella näitä rivejä löytyy koodista yli 900 kpl joten tämä sääntö muutos lienee parempi tehdä eri pullarissa joskus toiste.